ember-drag-drop/src/services/drag-coordinator.js

```diff
@@ -77,7 +77,7 @@ export default Service.extend({
   draggingOver(event, emberObject) {
     const currentOffsetItem = this.get('currentOffsetItem');
     const pos = this.relativeClientPosition(emberObject.element, event);
-    const hasSameSortingScope = this.get('currentDragItem.sortingScope') === emberObject.get('sortingScope');
```

Comment:
Why separate these? In the new version you'd get an error if the first object is undefined.